src/CoreFoundation/CFPropertyList.cs

```diff
@@ -14,10 +14,7 @@
 
 namespace CoreFoundation
 {
-	// This is currently incomplete and thus marked internal;
-	// it is only used by CFPreferences, and I'm not necessarily
-	// sure we'd want to expose it? -abock
-	internal class CFPropertyList : INativeObject, IDisposable
+	public class CFPropertyList : INativeObject, IDisposable
```

Comment:
This needs;

1. A review of the entire `CFPropertyList` API.
2. Tests for the entire  `CFPropertyList` API.
3. It needs a `CFPropertyList (IntPtr handle, bool owns)` constructor, and the `CFPropertyList (IntPtr)` constructor must call it with `owns=false` so that it matches the behaviour of other INativeObject types (this changes the existing behaviour, which also means all existing (internal) usages of `CFPropertyList` must be audited.

Reply:
WIth 1. I meant review everything _already bound_ (so that we don't make something public that's broken).

Reply:
I did (1)

Reply:
I've now done all of these.